src/builder/Actions.sol

```diff
@@ -122,6 +124,55 @@ library Actions {
         uint256 blockTimestamp;
     }
 
+    struct CometBorrowInput {
+        Accounts.ChainAccounts[] chainAccountsList;
+        uint256 amount;
+        string assetSymbol;
+        uint256 blockTimestamp;
+        address borrower;
+        uint256 chainId;
+        uint256[] collateralAmounts;
+        string[] collateralAssetSymbols;
+        address comet;
+    }
+
+    struct CometRepayInput {
+        Accounts.ChainAccounts[] chainAccountsList;
+        uint256 amount;
+        string assetSymbol;
+        uint256 blockTimestamp;
+        uint256 chainId;
+        uint256[] collateralAmounts;
+        string[] collateralAssetSymbols;
+        address comet;
+        address repayer;
+    }
+
+    struct MorphoBorrow {
+        Accounts.ChainAccounts[] chainAccountsList;
+        string assetSymbol;
+        uint256 amount;
+        uint256 chainId;
+        address borrower;
+        uint256 blockTimestamp;
+        uint256 collateralAmount;
+        string collateralAssetSymbol;
+    }
+
+    struct MorphoRepay {
+        Accounts.ChainAccounts[] chainAccountsList;
+        string assetSymbol;
+        uint256 amount;
+        uint256 chainId;
+        address repayer;
+        uint256 blockTimestamp;
+        uint256 collateralAmount;
+        string collateralAssetSymbol;
+        // Needed for handling max repay
+        bool payInShares;
+        uint256 borrowedShares;
```

Comment:
So I'm guessing we need this because there is no `max repay` in Morpho?

Reply:
Yes that's correct. 
For Morpho its doc say if you want to pay off all, use shares to pay off to prevent from having dust. 
```
It is advised to use the shares input when repaying the full position to avoid reverts due to conversion roundings between shares and assets.
```

Reply:
Just made this handled within MorphoScript, and leave builder not to worry about it.

Reply:
Cool, yeah that feels a cleaner and easier to manage.